web/client/components/app/StandardApp.jsx

```diff
@@ -55,9 +64,11 @@ class StandardApp extends React.Component {
                     require('intl/locale-data/jsonp/en.js');
                     require('intl/locale-data/jsonp/it.js');
                     this.init(config);
+                    this.addProjDefinitions(config);
```

Comment:
Can we move this into init, so that we don't need to call it in two different places?

Reply:
i'll put it in the init method then